source/isaaclab/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 ---------
 
+0.41.2 (2025-07-15)
+~~~~~~~~~~~~~~~~~~~
+
+Fixed
+^^^^^
+
+* Fixed IndexError in :meth:`isaaclab.envs.mdp.events.reset_joints_by_scale`,
+  :meth:`isaaclab.envs.mdp.events.reset_joints_by_offsets` by indexing joint positions by env_ids and joint_ids
```

Comment:
NIT: "indexing joint positions by env_ids and joint_ids" -- Does not make grammatical sense :)